pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -91,7 +91,7 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, in io.Reader, out i
 	cmd.Flags().StringSliceVarP(&o.Env, "env", "e", o.Env, "Specify key value pairs of environment variables to set for the build container.")
 	cmd.Flags().StringVar(&o.FromBuild, "from-build", o.FromBuild, "Specify the name of a build which should be re-run")
 
-	cmd.Flags().BoolVar(&o.Follow, "follow", o.Follow, "Start a build and watch its logs until it completes or fails")
+	cmd.Flags().BoolVarP(&o.Follow, "follow", "f", o.Follow, "Start a build and watch its logs until it completes or fails")
```

Comment:
`-f` is generally for file... I'd expect that to map to `--from-file`, if anything